kafka-connect-s3/docs/configuration_options.rst

```diff
@@ -148,12 +148,19 @@ S3
   * Importance: low
 
 ``s3.part.retries``
-  Number of upload retries of a single S3 part. Zero means no retries.
+  Maximum number of retry attempts for failed requests. Zero means no retries. An actual number of attempts is determined by the S3 client based on multiple factors, including, but not limited to - the value of this parameter, type of exception occurred, a number of attempts to retry failed request(s) etc.
```

Comment:
Thanks, fixed articles.

By last statements, I wanted to emphasize that the actual number of max attempts used in the S3 client can be lesser than the value of this parameter since S3 client tracks capacity of already retried failed requests and never goes above it even if `s3.part.retries` is bigger.
As a reference - https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/http/AmazonHttpClient.java#L1477.

By default, the S3 client has the capacity to retry failed request(s) 100 times, so there is a chance that we will exhaust that capacity earlier, therefore, I don't want users to be surprised in that case.

Reply:
I see. The sentence _a number of attempts to retry failed request(s) etc_ doesn't seem to make this clear. 
I'd suggest that we skip it, or say something like: 

_throttling settings of the underlying S3 client, etc_ 

since this is indeed controlled by the `com.amazonaws.sdk.enableThrottledRetry` property and what's used by default in the client we instantiate.

Reply:
Sure, let me reframe it a bit.